pages/apply/verify.tsx

```diff
@@ -30,16 +32,19 @@ const ApplicationVerifyPage = (): JSX.Element => {
 
   const confirmSignUp = async (values: FormData) => {
     try {
-      await Auth.confirmSignUp(emailAddress, values.code);
+      //await Auth.confirmSignUp(emailAddress, values.code);
```

Comment:
We should probably tidy up any references to AWS Cognito, deleting code like this and any unused functions.